signer/migrations/0003__create_tables.sql

```diff
@@ -184,3 +191,111 @@ CREATE TABLE sbtc_signer.withdrawal_reject_events (
     created_at    TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
 );
 
+-- Represents a combined transaction package which is broadcasted to the Bitcoin
+-- network. The transaction package is built up of multiple transactions which
+-- are tracked separately in the `sweep_transactions` table. A transaction
+-- package may contain the sweeping transactions for both deposit and
+-- withdrawal requests.
+CREATE TABLE sbtc_signer.sweep_packages (
+    -- Internal ID of the package
+    id SERIAL PRIMARY KEY,
```

Comment:
I don't think we should note things at the package level in our database. Although we construct and broadcast transactions as a package, they are not guaranteed to all be included or excluded for any given block. A bitcoin miner could include the first, say, 3 transactions from our "package" but not the following ones. For this reason tying them together might not be so helpful.

Reply:
Alright, gotcha, then I agree that this isn't needed from a functional perspective.

I'm still in favor of having this from a normalization perspective, though. The transactions in the package are all created at the same block hash, time and using the same market rate, which are three columns which then don't need to be stored separately on each transaction.

Let's discuss this one further on the call.

Reply:
Removed package-level storage in [25c1b2f](https://github.com/stacks-network/sbtc/pull/710/commits/25c1b2ffd730e593228cc57e12574671aa9530cd)